packages/strapi-plugin-content-type-builder/config/routes.json

```diff
@@ -0,0 +1,44 @@
+{
+  "routes": [
+    {
+      "method": "GET",
+      "path": "/models",
+      "handler": "ContentTypeBuilder.models",
+      "config": {
+        "policies": []
+      }
+    },
+    {
+      "method": "GET",
+      "path": "/models/:model",
+      "handler": "ContentTypeBuilder.model",
+      "config": {
+        "policies": []
+      }
+    },
+    {
+      "method": "POST",
+      "path": "/models",
+      "handler": "ContentTypeBuilder.create",
```

Comment:
Why this action name hasn't the same convention as the others? Should it be called `createModel` based on the `updateModel` and `deleteModel` methods?